From fd720131909a9f29568dab8a856ec030343fa4c0 Mon Sep 17 00:00:00 2001
From: Shinya Kuribayashi <shinya.kuribayashi@necel.com>
Date: Mon, 7 Sep 2009 09:45:58 +0900
Subject: [PATCH 15/15] MIPS: nec_emma3pf: Remove unused files / entries.

Recently updated nec_emma common parts obsolete these files / entries.
We no longer need them with respect to each EMMAXXX SoC.

Signed-off-by: Shinya Kuribayashi <shinya.kuribayashi@necel.com>
---
 arch/mips/Makefile                  |    6 ------
 include/asm-mips/mach-emma3pf/irq.h |   24 ------------------------
 include/asm-mips/mach-emma3pf/war.h |   25 -------------------------
 3 files changed, 0 insertions(+), 55 deletions(-)
 delete mode 100644 include/asm-mips/mach-emma3pf/irq.h
 delete mode 100644 include/asm-mips/mach-emma3pf/war.h

diff --git a/arch/mips/Makefile b/arch/mips/Makefile
index 443afb8..ac6ddc0 100644
--- a/arch/mips/Makefile
+++ b/arch/mips/Makefile
@@ -413,12 +413,6 @@ core-$(CONFIG_MARKEINS)         += arch/mips/emma2rh/markeins/
 load-$(CONFIG_MARKEINS)         += 0xffffffff88100000
 
 #
-# NEC EMMA3PF boards
-#
-core-$(CONFIG_SOC_EMMA)		+= arch/mips/emma/common/
-cflags-$(CONFIG_SOC_EMMA3PF)	+= -Iinclude/asm-mips/mach-emma3pf
-
-#
 # NEC EMMA3PF ET-MC10121
 #
 core-$(CONFIG_NEC_ET10121)	+= arch/mips/emma/et10121/
diff --git a/include/asm-mips/mach-emma3pf/irq.h b/include/asm-mips/mach-emma3pf/irq.h
deleted file mode 100644
index 35d8e9a..0000000
--- a/include/asm-mips/mach-emma3pf/irq.h
+++ /dev/null
@@ -1,24 +0,0 @@
-/*
- *  This program is free software; you can redistribute it and/or modify
- *  it under the terms of the GNU General Public License as published by
- *  the Free Software Foundation; either version 2 of the License, or
- *  (at your option) any later version.
- *
- *  This program is distributed in the hope that it will be useful,
- *  but WITHOUT ANY WARRANTY; without even the implied warranty of
- *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
- *  GNU General Public License for more details.
- *
- *  You should have received a copy of the GNU General Public License
- *  along with this program; if not, write to the Free Software
- *  Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA  02111-1307  USA
- *
- */
-#ifndef __ASM_MACH_EMMA_IRQ_H
-#define __ASM_MACH_EMMA_IRQ_H
-
-#define NR_IRQS	256
-
-#include_next <irq.h>
-
-#endif
diff --git a/include/asm-mips/mach-emma3pf/war.h b/include/asm-mips/mach-emma3pf/war.h
deleted file mode 100644
index e0941fb..0000000
--- a/include/asm-mips/mach-emma3pf/war.h
+++ /dev/null
@@ -1,25 +0,0 @@
-/*
- * This file is subject to the terms and conditions of the GNU General Public
- * License.  See the file "COPYING" in the main directory of this archive
- * for more details.
- *
- * Copyright (C) 2002, 2004, 2007 by Ralf Baechle <ralf@linux-mips.org>
- */
-#ifndef __ASM_MIPS_MACH_EMMA3PF_WAR_H
-#define __ASM_MIPS_MACH_EMMA3PF_WAR_H
-
-#define R4600_V1_INDEX_ICACHEOP_WAR	0
-#define R4600_V1_HIT_CACHEOP_WAR	0
-#define R4600_V2_HIT_CACHEOP_WAR	0
-#define R5432_CP0_INTERRUPT_WAR		0
-#define BCM1250_M3_WAR			0
-#define SIBYTE_1956_WAR			0
-#define MIPS4K_ICACHE_REFILL_WAR	0
-#define MIPS_CACHE_SYNC_WAR		0
-#define TX49XX_ICACHE_INDEX_INV_WAR	0
-#define RM9000_CDEX_SMP_WAR		0
-#define ICACHE_REFILLS_WORKAROUND_WAR	0
-#define R10000_LLSC_WAR			0
-#define MIPS34K_MISSED_ITLB_WAR		0
-
-#endif /* __ASM_MIPS_MACH_EMMA3PF_WAR_H */
-- 
1.6.3.3

